index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Portfolio Website</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/69bb18a36d.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="flexbox">
+        <span class="logo">Mudasiri</span>
+        <nav>
+            <i class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <section class="intro">
+        <h1 class="title">I’m Mudasir
+            Welcome to my Site!</h1>
+        <p class="description">I’m a fullstack web developer! I can help you build a product website, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3 class="action-text">LET’S CONNECT</h3>
+        <ul class="social-media">
+            <li><i class="fa-brands fa-twitter"></i></li>
+            <li><i class="fa-brands fa-linkedin"></i></li>
+            <li><i class="fa-brands fa-github"></i></li>
+            <li><i class="fa-brands fa-medium"></i></li>
```

Comment:
- [ ] Kindly make your list items the links  (i.e `a` tag) to your social media pages. This is for the semantics reason and accessibility purpose